### ISSUE TITLE
Allow multiple articles to be opened

### ISSUE THREAD

**Issue by radiovisual** [crypti/ccnews#1]
I find that more often than not, I want to open up more than one article when I see the list of options, and so it feels overkill to run the command a second or third time just to open a second or third article.

So I propose that either we:

1. allow users to enter a comma-delimited list of articles to open, or 
2. keep the cli open, waiting for input until the user explicitly exits

The pro/con of #2 is that is will only exit when the user decides to exit, but that means they can open all the articles they want before they are done, and the con of #1 is that even though it allows multiple articles to be opened, ccnews would have to be run again with the same options if the user decides they want to read an article that they forgot to select.

@brh55, thoughts?

**Comment by radiovisual** [comment]
Another option is to print out the url (with an option, maybe?), and savvy users can open the article straight from the terminal.

**Comment by brh55** [comment]
Good point. Well we could provide the ability to do all of these options. 
- Leave the CLI open
- Provide an additional link to be clicked 
- Allow a list of articles to be opened

**Comment by radiovisual** [comment]
Yeah, true. I like the idea of entering into a "news mode" in the cli, where I have to explicitly exit, its not that uncommon for my processes to enter into a mode where I need to run a command to exit, so it will feel invasive. I like it. 👍

**Comment by brh55** [comment]
I concur :hammer:. Going to do that :D
